Examples/Swift/OfflineViewController.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+import Mapbox
+import MapboxDirections
+import MapboxCoreNavigation
+
+class OfflineViewController: UIViewController {
+    
+    var mapView: MGLMapView!
+    var resizableView: ResizableView!
+    var backgroundLayer = CAShapeLayer()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView = MGLMapView(frame: view.bounds)
+        view.addSubview(mapView)
+        
+        backgroundLayer.frame = view.bounds
+        backgroundLayer.fillColor = #colorLiteral(red: 0.1450980392, green: 0.2588235294, blue: 0.3725490196, alpha: 0.196852993).cgColor
+        view.layer.addSublayer(backgroundLayer)
+        
+        resizableView = ResizableView(frame: CGRect(origin: view.center,
```

Comment:
Nit: This is line-broken in a weird way.

Reply:
fixed